### ISSUE TITLE
Geef aan of foto / beschrijving gewenst is

### ISSUE THREAD

**Issue by Desharin** [Desharin/VERA-issues#52]
Bij waarnemingen van bepaalde soorten zijn foto's of beschrijvingen gewenst. Dit zou kunnen worden aangegeven wanneer deze soort gekozen wordt.

De makkelijkste implementatie is waarschijnlijk door gebruik te maken van de species_message.
Anders moeten er duidelijke criteria worden opgesteld wanneer het bericht getoond moet worden.

**Comment by laurenssparrius** [comment]
Ja, species_message is de plek hiervoor. Punt is dat we voor planten geen lijst hebben van soorten waarvoor we foto's vereisen. We gaan er vanuit dat waarnemers een foto meesturen als ze vermoeden dat de waarneming niet door de validatie heen komt, of als ze de soortnaam niet zeker weten.

**Comment by laurenssparrius** [comment]
Is of wordt opgelost in de Verspreidingsatlas
